bbc/simorgh
# Add local TIPO homepages to all single language services - WSTEAMA-381

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAMA-381

Overall changes
======
- Adds local mock `simorgh-bff` responses for new `Home` pages where services have no variants
- All services except for `igbo` use TIPO live data. `igbo` uses test as live has not yet been set up

**Local development environment paths for** `Home`:

http://localhost:7080/afaanoromoo
http://localhost:7080/afrique
http://localhost:7080/amharic
http://localhost:7080/arabic
http://localhost:7080/azeri
http://localhost:7080/bengali
http://localhost:7080/burmese
http://localhost:7080/gahuza
http://localhost:7080/gujarati
http://localhost:7080/hausa
http://localhost:7080/hindi
http://localhost:7080/igbo
http://localhost:7080/indonesia
http://localhost:7080/japanese
http://localhost:7080/korean
http://localhost:7080/marathi
http://localhost:7080/mundo
http://localhost:7080/nepali
http://localhost:7080/pashto
http://localhost:7080/persian
http://localhost:7080/pidgin
http://localhost:7080/portuguese
http://localhost:7080/punjabi
http://localhost:7080/russian
http://localhost:7080/sinhala
http://localhost:7080/somali
http://localhost:7080/swahili
http://localhost:7080/tamil
http://localhost:7080/telugu
http://localhost:7080/thai
http://localhost:7080/tigrinya
http://localhost:7080/turkce
http://localhost:7080/ukrainian
http://localhost:7080/urdu
http://localhost:7080/vietnamese
http://localhost:7080/yoruba

Code changes
======

- Loads of new .json files...

Testing
======
1. Go to `http://localhost:7080/{service}` where `service` is a single script service
2. See the `Home` page load in the browser
3. Additional verification can be achieved by appending url with `.json` and viewing the `dataSource` key validating where the response body originates

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
